client/src/components/Image.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { withStyles } from '@material-ui/core'
+import DeleteOutlinedIcon from '@material-ui/icons/DeleteOutlined';
+
+const styles = {
+  deleteIconContainer: {
+    position: 'absolute',
+    height: '75px',
+    width: '75px',
+    backgroundColor: 'grey',
+    opacity: '50%',
+    borderRadius: '10px',
```

Comment:
Same as above, any px can simply be put as a js number, I think it processes it as a px unit by default, so that will probably apply to any values that are similar in the following code.